### PR TITLE
Streamline root PrimusConf

### PR DESCRIPTION
**Pull request by hopanghsu-bytedance**
PrimusConf is the comprehensive configuration for a Primus application, which inevitably contains many settings. Hence, to improve user experience, this merge request focuses on cleaning up the root PrimusConf by grouping them and deprecating stale features.

### CODE REVIEW

File: primus-core/src/main/java/com/bytedance/primus/executor/worker/launchplugin/EnvPlugin.java [hunk 1]
```diff
@@ -24,21 +24,16 @@
...
-import com.bytedance.primus.proto.PrimusConfOuterClass.PrimusConf;
 import java.util.HashMap;
 import java.util.Map;
 import org.apache.commons.lang.StringUtils;
 
 public class EnvPlugin implements ChildLaunchPlugin {
 
-  private static final String APPLICATION_ID_ENV_KEY = "APPLICATION_ID";
-  private static final String CONTAINER_ID_ENV_KEY = "CONTAINER_ID";
   private static final String ORACLE_ID_ENV_KEY = "ORACLE_ID";
```
Comment (qianhan666) [inline primus-core/src/main/java/com/bytedance/primus/executor/worker/launchplugin/EnvPlugin.java:35]: Can we change the name of env key to EXECUTOR_ID

Comment (hopanghsu-bytedance) [reply primus-core/src/main/java/com/bytedance/primus/executor/worker/launchplugin/EnvPlugin.java:35]: Sorry, I don't quite understand, aren't they removed?

Comment (qianhan666) [reply primus-core/src/main/java/com/bytedance/primus/executor/worker/launchplugin/EnvPlugin.java:35]: ORACLE_ID is used by an internal component. For tf or pytorch, maybe we can use EXECUTOR_ID

Comment (hopanghsu-bytedance) [reply primus-core/src/main/java/com/bytedance/primus/executor/worker/launchplugin/EnvPlugin.java]: done :D

**Review by misakamisaka — APPROVED** [review]

**Review by qianhan666 — APPROVED** [review]

**Pull request merged**